plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -4881,6 +4881,35 @@ public void testDropColumn()
         assertUpdate("DROP TABLE test_drop_column");
     }
 
+    @Override
+    public void testDropAndAddColumnWithSameName()
```

Comment:
Since in case Hive kind of gives invalid result for dropping and creating a column with same name do we need to test it. If we use `transactionalTable` does this solve this issue ?

Reply:
Even transaction table has the same issue.